SofaKernel/framework/sofa/helper/system/DynamicLibrary.cpp

```diff
@@ -63,7 +64,9 @@ const std::string& DynamicLibrary::Handle::filename() const
 DynamicLibrary::Handle DynamicLibrary::load(const std::string& filename)
 {
 # if defined(WIN32)
-    void *handle = ::LoadLibraryA(filename.c_str());
+    std::string p(filename);
+    std::replace(p.begin(), p.end(), '/', '\\'); // ensure Windows style path
```

Comment:
is this still really needed in 2019 ?

Reply:
I see you define this old trick as well in cleanPath. Maybe will be more consistent to call 
`LoadLibraryExA(cleanPath(p).c_str(), ...`

Reply:
I followed your advice but also deprecated FileRepository::cleanPath as it basically was a duplicate of FileSystem::cleanPath.